pipelines/run-generic-pipelines-on-kubeflow-pipelines/README.md

```diff
@@ -0,0 +1,286 @@
+
+   ![Customize container resources](doc/images/customize-resources.png)
+
+   > If no custom resource requirements are defined, the defaults in the Kubeflow Pipeline environment are used.
+
+   Containers in which the notebooks or scripts are executed don't share a file system. Elyra utilizes S3-compatible cloud storage to facilitate the transfer of files from the JupyterLab environment to the containers and between containers.
+
+   ![Files are exchanged using S3 storage](doc/images/file-inputs-and-outputs.png)
+
+   Therefore you must declare files that the notebook or script requires and declare files that are being produced. The node you are inspecting does not have any file input dependecies but it does produce an output file.
+
+   ![Declare file inputs and outputs](doc/images/file-io-declarations.png)
+
+   Notebooks and scripts can be parameterized using environment variables. The node you are looking at requires a variable that identifies the download location of a data file.
+
+   ![Declare environment variables](doc/images/environment-variable-declarations.png)
+
+   Refer to [_Best practices for file-based pipeline nodes_](https://elyra.readthedocs.io/en/stable/user_guide/best-practices-file-based-nodes.html) in the _User Guide_ to learn more about considerations for each configuration setting.
+
+### Define a runtime environment configuration
+
+A [runtime configuration](https://elyra.readthedocs.io/en/stable/user_guide/runtime-conf.html) in Elyra contains connectivity information for a Kubeflow Pipelines instance and S3-compatible cloud storage. In this tutorial you will use the GUI to define theconfiguration, but you can also use the CLI.
+
+1. From the pipeline editor tool bar (or the JupyterLab sidebar on the left side) choose _Runtimes_ to open the runtime management panel.
+
+   ![Open runtime configuration panel](doc/images/open-runtime-configuration-panel.png)
+
+1. Click **+** and _New Kubeflow Pipelines runtime_ to create a new configuration for your Kubeflow Pipelines deployment.
+
+1. Enter a _name_ and a _description_ for the configuration and optionally assign _tags_ to support searching.
+
+   ![Define misc properties](doc/images/define-misc-properties.png)
+
+1. Enter the connectivity information for your Kubeflows Pipelines deployment:
+   - _Kubeflow Pipelines API endpoint_, e.g. `https://kubernetes-service.ibm.com/pipeline`
+     > Do not specify the namespace in the API endpoint.
+   - _User namespace_ used to run pipelines, e.g. `mynamespace`
+   - User credentials if the deployment is multi-user, auth enabled using [Dex](https://www.kubeflow.org/docs/started/k8s/kfctl-istio-dex/).
+   - Workflow _engine_ type, which is either `Argo` (default) or `Tekton`. Check with an administrator if you are unsure which workflow engine your deployment utilizes.
+
```

Comment:
This link is broken for me, and I don't think I know enough about this subject that I can hunt down the correct new link

Reply:
Removing link for now, can't find a good resource ...